Source/Libraries/NetCore/NetcoreCommands.cs

```diff
@@ -1,138 +1,129 @@
-namespace RTCV.NetCore
```

Comment:
Hey @NarryG , this change is going to be pretty substantial. I wanted to show you part of it and get your thoughts before completing it.

Reply:
That's fine. The commands are somewhat inconsistent in styling so feel free to make them more consistent while you're at it.

Reply:
I don't know what to do with melonDS for this change: 

https://github.com/redscientistlabs/melonDS-Vanguard/blob/Vanguard/src/Vanguard/VanguardClient.cpp#L833

Reply:
I'm not entirely sure what to do about that either. In the end, the message is just a string. While it's operating on System::String, it's still primarily limited by the rules of C++. Since there's no native support for string switches over on the C++ side, I wrote that little wrapper so I didn't have a huge if-else chain.  If you have any ideas I'm up for suggestions.

Reply:
I don't have anything off the top of my head, but let me think about it. I'll respond back to this thread when/if I come up with something

Reply:
I took another look at this. I think a massive if-else chain is actually the best course of action here. There's a lot of different hacks in C++ to try to get it to be a competent language when it comes to switching strings. Unfortunately, there's no solution that won't involve us breaking the DRY principle, or introducing unneeded complexity,  since our strings are coming from C# land.

Reply:
These aren't iterated at an amount where it's a massive performance concern so I guess as long as it's readable, it's fine.